Gruntfile.js

```diff
@@ -77,20 +77,14 @@ grunt.initConfig({
         spawn: false
       }
     }
-  },
-  templateSize: {
-    benchmark: {
-      src: ['benchmark/templates/*.td']
-    }
   }
 });
 
-grunt.task.loadTasks('grunt-tasks/');
 grunt.loadNpmTasks('grunt-mocha');
 
 grunt.registerTask('dist', ['peg', 'eslint', 'babel:dist']);
 grunt.registerTask('test', ['dist', 'babel:acceptance', 'browserify:test', 'mocha']);
 grunt.registerTask('sandbox', ['dist', 'browserify:sandbox']);
-grunt.registerTask('benchmark', ['dist', 'bytesize', 'templateSize']);
+grunt.registerTask('benchmark', ['dist', 'bytesize']);
```

Comment:
I'm just going to delete the file, then. It was breaking the grunt task, because `dist` is no longer available until after `grunt` finishes.